app/models/post.rb

```diff
@@ -1,21 +1,19 @@
 class Post < ApplicationRecord
-  validates :title, presence: true, length: { maximum: 250 }
-  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
-  #  attr_accessor :text, :title, :content, :comments_counter, :likes_counter
-
-  belongs_to :author, foreign_key: 'author_id', class_name: 'User'
+  belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
 
-  after_save :update_author_posts_count
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { greater_than_or_equal_to: 0 }
 
```

Comment:
- [ ] You're almost there but I encountered an issue when creating a post from the rails console and after debugging for a couple of minutes I noticed the issue was from the validation to the comments_counter and likes_counter you applied keep in mind a post can be created without specifying the comments_counter and likes_counter value so please make sure to fix this (you can give it an intial value of 0).